pandas/io/clipboards.py

```diff
@@ -108,8 +112,9 @@ def to_clipboard(obj, excel=True, sep=None, **kwargs):  # pragma: no cover
                 text = text.decode('utf-8')
             clipboard_set(text)
             return
-        except:
-            pass
+        except TypeError:
+            warnings.warn('to_clipboard in excel mode requires a single \
+            character separator. Set "excel=False" or change the separator')
```

Comment:
Can you use implicit line continuation instead of `\` ?

Like
```
            warnings.warn('to_clipboard in excel mode requires a single '
                          'character separator. Set "excel=False" or change '
                          'the separator')
```